### ISSUE TITLE
Type error when using cv.findContours

### ISSUE THREAD

**Issue by backrunner** [TechStark/opencv-js#34]
```ts
const contours = new cv.MatVector();
const hierarchy = new cv.Mat();
cv.findContours(image, contours, hierarchy, cv.RETR_LIST, cv.CHAIN_APPROX_SIMPLE);
```
After the `cv.findContours` executed, `contours.size()` will be typed as `cv.Size`, it should be a number.

**Comment by ewwwgiddings** [comment]
I think this may be a work around for right now? `(contours.size() as any as number)`

**Comment by backrunner** [comment]
> I think this may be a work around for right now? `(contours.size() as any as number)`

That's exactly what I'm doing now,  but the type issue is still painful🥲

**Comment by neko-para** [comment]
Seems that mirada is not maintained now 😢 . Does anyone know any alternative?

**Comment by ttt43ttt** [comment]
The plan is moving the typescript definitions from mirada to this repo. We’ll maintain them here. This is in progress.

**Comment by ttt43ttt** [comment]
Fixed in v4.8.0-release.6
